docs/topics/native/native-objc-interop.md

```diff
@@ -32,6 +32,10 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a framewor
 * See [Build final native binaries](multiplatform-build-native-binaries.md#declare-binaries) to see how to declare binaries.
 * Check out the [Kotlin Multiplatform sample project](https://github.com/Kotlin/kmm-basic-sample) for an example.
 
+### Create bindings for a new library
+
+To create bindings for a new library, first create and configure a [definition file](native-definition-file.md).
```

Comment:
The whole task started with a complaint that the def file is not mentioned on the Obj-C interop page at all. Together with @sbogolepov we decided that it makes sense to create a separate page for .def file instead of duplicating the information on both C interop and  Obj-C interop pages

I've added a new header to highlight that that's a separate case and the list of links to both pages. Hope it'll make them more visible.